commands/hubspot.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../base_command'
+require "mechanize"
+require 'nokogiri'
+require 'open-uri'
+require 'pry'
+class Hubspot < BaseCommand
+  def perform
+    agent = Mechanize.new
+    page = agent.get('https://app.fooda.com/login')
+
+    # login
+    form = agent.page.forms.first
+    form.fields[2].value = 'food@food.com'
+    form.fields[3].value = 'Pursu1ts'
```

Comment:
David - 

Have a look at the `sample.env` files, and add keys for the fooda credentials (but not the real values).

You'll then also add the proper credentials in the Heroku config either via command line or in the dashboard in Settings.

You can do

`ENV['key']`

or 

`ENV.fetch('key')` 

The latter will fail when the app loads if it cannot location the variable.

Note: `ENV.fetch('key', default)`  is another option.